src/lib/extend.ts

```diff
@@ -150,10 +150,7 @@ const legacyExtend = {
 };
 
 const modernExtend = {
-    lightBrightnessColortempColor: (args: {
-        disableEffect: boolean, supportsHueAndSaturation: boolean, disableColorTempStartup: boolean, preferHueAndSaturation: boolean,
-        disablePowerOnBehavior: boolean,
-    }): ModernExtend => {
+    lightBrightnessColortempColor: (args: Extend.options_light_onoff_brightness_colortemp_color={}): ModernExtend => {
```

Comment:
I didn't use this on purpose since it also includes `fromZigbee`, `toZigbee` and `exposes` which we don't want (that should be another extend)